src/main/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepository.java

```diff
@@ -86,7 +86,10 @@ public UUID insert(NewEnvelope envelope) {
                 .addValue("fileName", envelope.fileName)
                 .addValue("fileCreatedAt", Timestamp.from(envelope.fileCreatedAt))
                 .addValue("status", envelope.status.name())
-                .addValue("dispatchedAt", Timestamp.from(envelope.dispatchedAt))
+                .addValue(
+                    "dispatchedAt",
+                    envelope.dispatchedAt == null ? null : Timestamp.from(envelope.dispatchedAt)
+                )
```

Comment:
We should have some tests around it. Perhaps it would be best to extract this bit into a separate PR then?